src/v0/destinations/user/utils.js

```diff
@@ -277,6 +278,7 @@ const updateCompany = async (message, destination, company) => {
   const response = await httpPUT(endpoint, payload, requestOptions, {
     destType: 'user',
     feature: 'transformation',
+    endpointPath: `/companies/`,
```

Comment:
We should probably include `httpMethod` as well because in this case, `/companies` API is being used in POST & PUT http calls but as per stats we will only have a single endpointPath(`/companies/`) & not be able to differentiate between the POST / PUT calls.

Reply:
Else in `endpointPath` include PUT / POST as a prefix or suffix

Reply:
cc: @koladilip @yashasvibajpai

Reply:
Do you mean to add this specifically for user destination, or for all destinations across?

Reply:
@sanpj2292 does API limits differ based on PUT or POST method? if we decided to do then @yashasvibajpai better to do all destinations as it will be consistent

Reply:
@koladilip The destinations' implementations have exclusively one out of POST or PUT endpoints, so they don't have different limits for different request methods. 
on the structure, does this suffice?
for eg. like below
"[METHOD]/. . . /companies/"

Reply:
in that case I am fine either ways adding or not adding both are fine, if we add better to add as dedicated label rather than appending to endPoint label

Reply:
If limits is the use-case for which we are adding `endpointPath` label & if limits per http methods doesn't change, it is fine to proceed without adding another label or prefixing

Reply:
Yes, majorly those were the reasons. Not adding then

Reply:
Cool. Resolving the comment then